sympy/solvers/solvers.py

```diff
@@ -2453,24 +2482,32 @@ def solve_linear_system(system, *symbols, **flags):
 
 
 def solve_undetermined_coeffs(equ, coeffs, sym, **flags):
-    """Solve equation of a type p(x; a_1, ..., a_k) == q(x) where both
-       p, q are univariate polynomials and f depends on k parameters.
-       The result of this functions is a dictionary with symbolic
-       values of those parameters with respect to coefficients in q.
+    """
```

Comment:
There needs to be an `r` here before the quotes. This is causing the tests to fail. Any docstring with backslashes has to use `r"""`. I don't remember if we put this in the style guide.

Reply:
Yes we did, I just missed this. Thanks!